.build/automation/stages/localization-handback.yml

```diff
@@ -0,0 +1,85 @@
+
+stages:
+  - stage: localization_handback
+    displayName: Localization Handback
+    dependsOn: []
+    condition: and(succeeded(), eq(variables.isLocBranch, true))
+
+    jobs:
+      - job : generate_resx
+        displayName: 'Process incoming translations'
+        pool:  $(HostedWinVS2019)
+        steps:
+          - checkout: self
+            persistCredentials: true
+            clean: true
+
+          - powershell: |
+              #-- Import Invoke-Git Module function
+              Import-Module $(Build.SourcesDirectory)\.build\automation\scripts\Invoke-Git.psm1 -Force -DisableNameChecking
+              git config user.email "vs-mobiletools-engineering-service2@microsoft.com"
+              git config user.name "VS Mobile Engineering Serice Account"
+              Invoke-Git checkout main
+              Invoke-Git merge origin/loc --no-commit
+            displayName: 'Merge loc Branch'
+
+          - task: cesve.one-loc-build.one-loc-build.OneLocBuild@2
+            displayName: 'Localization Build'
+            env:
+              SYSTEM_ACCESSTOKEN: $(System.AccessToken) 
+            inputs:
+              locProj: '.config/LocProject.json'
+              outDir: '$(Build.ArtifactStagingDirectory)'
+              packageSourceAuth: patAuth
+              patVariable: "$(OneLocBuildPAT)"
+              isCreatePrSelected: false
+              repoType: gitHub
+              prSourceBranchPrefix: $(LocBranchPrefix)
+              gitHubPatVariable: '$(GitHub.Token)'
+              gitHubPrMergeMethod: merge
+
+          - powershell: |
+              #-- Import Invoke-Git Module function
+              Import-Module $(Build.SourcesDirectory)\.build\automation\scripts\Invoke-Git.psm1 -Force -DisableNameChecking
+              #--Clear Stage
+              Invoke-Git reset HEAD -- .
+              #-- Create new branch
+              $branchName = "$(LocBranchPrefix)/$(new-guid)"
+              Invoke-Git checkout -B ${branchName}
+              Write-Host ("##vso[task.setvariable variable=HANDBACK_BRANCH_NAME;]$branchName")
+              #-- Stage Build Changes
+              Invoke-Git add Designer
```

Comment:
I don't think this would work since those folders are not on the source repo, right?